src/demo/share/jfc/SwingSet2/resources/swingset_ja.properties

```diff
@@ -451,7 +451,7 @@ SliderDemo.accessible_description=\u3053\u306E\u30C7\u30E2\u3067\u306F\u3001JSli
 SliderDemo.tooltip=JSlider\u30C7\u30E2
 SliderDemo.name=Slider\u30C7\u30E2
 
-SliderDemo.slidervalue=Slider\u306E\u5024: 
+SliderDemo.slidervalue=Slider\u306E\u5024:\u0020
```

Comment:
Unsure. It's used in four places. The two that use it concatenated should probably have a space added in the code and this removed.